samza-core/src/main/java/org/apache/samza/task/StreamOperatorTask.java

```diff
@@ -134,7 +131,7 @@ public final void window(MessageCollector collector, TaskCoordinator coordinator
   }
 
   @Override
-  public void close() throws Exception {
+  public final void close() throws Exception {
```

Comment:
I'm not sure why some were final and some were non-final. I guess there was a reason to prevent an override from happening on these methods.
It looks like these got synced by someone else, so this change is unnecessary.